### ISSUE TITLE
[Bug] flock.js expects index.js.map to load, Chrome has difficulty

### ISSUE THREAD

**Issue by grempe** [tinybirdco/web-analytics-starter-kit#24]
### Module

Tracker

### Describe the Bug

Chrome console, when accessing flock.js, throws a warning about the missing `index.js.map` file.

```txt
DevTools failed to load source map: Could not load content for https://unpkg.com/@tinybirdco/index.js.map: HTTP error: status code 404, net::ERR_HTTP_RESPONSE_CODE_FAILURE
```
The `flock.js` file that `unpkg` points to does in fact point to a:

```javascript
...
//# sourceMappingURL=index.js.map
```

But if you ask for that directly it fails:

```
https://unpkg.com/@tinybirdco/index.js.map
Cannot find package @tinybirdco/index.js.map@latest
```

But looking for it after unpkg redirects it seems to exist.

Both OK

```
https://unpkg.com/@tinybirdco/flock.js@1.2.0/dist/index.js
https://unpkg.com/@tinybirdco/flock.js@1.2.0/dist/index.js.map
```

In my Next.js `_document.tsx` file I am using the following loading script:

```typescript
        <body>
          <Main />
          <NextScript />
          {/* https://www.tinybird.co/starter-kits/web-analytics : EU Data Store */}
          {process.env.NEXT_PUBLIC_VERCEL_ENV !== 'development' && (
            <Script
              src="https://unpkg.com/@tinybirdco/flock.js"
              data-host="https://api.tinybird.co"
              data-token={process.env.NEXT_PUBLIC_TINYBIRD_TRACKER_TOKEN}
              strategy="afterInteractive"
            ></Script>
          )}
        </body>
```

PS - I am filing this here since I was unable to find the source repository in your Github org that is the source for the `flock.js` tracker script.

### Expected Behavior

Warnings would not show in chrome console.

### To Reproduce

Load `flock.js` in Chrome browser.

### Details

Chrome Version 105.0.5195.125 (Official Build) (x86_64)

**Comment by alejandromav** [comment]
Hi @grempe! 

First of all, **thanks a lot for testing the Starter Kit**.

And yes, this is the right place to open an issue, since the source of `flock.js` is in this repository: https://github.com/tinybirdco/web-analytics-starter-kit/blob/main/middleware/src/index.js

Regarding you question, that's indeed an issue with [UNPKG](https://unpkg.com/) when using short URLs that omit the actual release version of the script (`@latest`).

For now, I can suggest you using this url as a workaround:

```js
<body>
    <Main />
    <NextScript />
    {/* https://www.tinybird.co/starter-kits/web-analytics : EU Data Store */}
    {process.env.NEXT_PUBLIC_VERCEL_ENV !== 'development' && (
        <Script
            src="https://unpkg.com/@tinybirdco/flock.js/dist/index.js"
            data-host="https://api.tinybird.co"
            data-token={process.env.NEXT_PUBLIC_TINYBIRD_TRACKER_TOKEN}
            strategy="afterInteractive"
        ></Script>
    )}
</body>
```

We are working on a better solution, I'll let you know as soon as we have one.

Please let me know if that works for you, you can email me at alejandromav@tinybird.co if you need any help or further assistance.

We'll keep you posted on this issue.
Thanks again, and sorry for the inconvenience! 🙌 🙏

**Comment by alejandromav** [comment]
This should be solved now with version 1.2.1. Make sure you're loading the latest version, it may take a while due to caching at many levels.

![image](https://user-images.githubusercontent.com/3041948/192572009-ce48d7fd-c753-491a-9828-90d2d642543d.png)

Let us know if the issue persists.
